extensions-core/kinesis-indexing-service/src/main/java/org/apache/druid/indexing/kinesis/supervisor/KinesisSupervisorIOConfig.java

```diff
@@ -91,7 +92,7 @@ public KinesisSupervisorIOConfig(
         earlyMessageRejectionPeriod,
         autoScalerConfig,
         lateMessageRejectionStartDateTime,
-        null
+        new IdleConfig(false, null)
```

Comment:
We should either pass both as null or both as the default values (`false` and `600_000`). I prefer both null as they are handled inside the `IdleConfig` anyway.